linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/rule.yml

```diff
@@ -52,3 +52,8 @@ ocil: |-
     return a line, or the line is commented out, this is a finding.
 
 platform: machine
+
+template:
+    name: audit_rules_privileged_commands
```

Comment:
The rule description needs an update to reflect that the RHEL rule goes with the `-F path=<PATH>` approach, instead of  `-w PATH`.

Reply:
all right

Reply:
Done in: https://github.com/ComplianceAsCode/content/pull/6910/commits/532417e9cf815077950991bbf764bc19a9d2510e#diff-da551e6bd2f13eaf859be2929f41f9a7c6bbed8022b4f968b997683ce006a9c5